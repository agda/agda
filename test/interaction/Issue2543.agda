-- Andreas, 2017-04-11, Issue 2543
-- Printing of second non-trivial with-pattern

-- {-# OPTIONS -v interaction.case:100 #-}

data D : Set where
  c : D → D

f : D → D
f y with c y
... | c z with c y
... | q = {!q!}  -- C-c C-c q

-- Was:
-- f y | c z | (c q) = ?

-- Expected:
-- f y | c z | c q = ?

-- Jesper, 2019-11-21, now:
-- ... | c q = ?
