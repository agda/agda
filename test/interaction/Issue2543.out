(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : D " nil)
((last . 1) . (agda2-goals-action '(0)))
((last . 2) . (agda2-make-case-action '("... | c q = ?")))
((last . 1) . (agda2-goals-action '(0)))
